src/components/Followers/FollowerDetails/FollowerDetails.js

```diff
@@ -22,12 +22,12 @@ const FollowerDetails = ({id, followersVisible}) => {
   }
 
   return (
-    <div className="profile-link">
+    <div className={!type ? 'profile-link' : 'main-link'}>
```

Comment:
I like the ternary set up for the different follower details, based on their type!